aws/instrumentation_test.go

```diff
@@ -109,6 +119,10 @@ type mockSQS struct {
 
 	spiedDeleteMessageInput   *sqs.DeleteMessageInput
 	stubbedDeleteMessageError error
+
+	spiedDeleteQueueInput   *sqs.DeleteQueueInput
+	spiedDeleteQueueOutput  *sqs.DeleteQueueOutput
```

Comment:
same here as with the `mockSNS`: just remove this if it's not used

Reply:
Using in DeleteSubscription

Reply:
you're not using `spiedDeleteQueueOutput` anywhere. It's not even something you should `spy` -- it's something you should `stub`. And in your `DeleteSubscription` call, you're not using it. That's why I said you should remove that field.